common/src/main/java/org/apache/uniffle/common/util/JavaUtils.java

```diff
@@ -22,13 +22,26 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.function.Function;
 
+import com.google.common.base.Enums;
 import org.apache.commons.lang3.JavaVersion;
 import org.apache.commons.lang3.SystemUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class JavaUtils {
   private static final Logger logger = LoggerFactory.getLogger(JavaUtils.class);
+  private static final String JAVA_9 = "JAVA_9";
+
+  public static boolean isJavaVersionAtLeastJava9() {
+    if (Enums.getIfPresent(JavaVersion.class, JAVA_9).isPresent()
```

Comment:
Why we should two checkings rather than 1?

Reply:
![image](https://github.com/apache/incubator-uniffle/assets/9440617/fc86060a-c8a3-47d4-853e-cb893f14bdf6)

for fix this error .

Reply:
`Enums.getIfPresent(JavaVersion.class, JAVA_9).isPresent()
`

is check is "JAVA_9" in enum JavaVersion.


`SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9)
`
is check is system support jdk9